TestResultSummaryService/parsers/Parser.js

```diff
@@ -7,7 +7,7 @@ class Parser {
 
     exactJavaVersion(output) {
         const javaVersionRegex = /(((openjdk|java) version[\s\S]*?)(JCL.*)|((openjdk|java) version[\s\S]*?)(Server VM.*))/;
-        const javaBuildDateRegex = /-(20[0-9][0-9][0-9][0-9][0-9][0-9])/;
+        const javaBuildDateRegex = /Compressed References\s(20[0-9][0-9][0-9][0-9][0-9][0-9])/gm;

```

Comment:
We cannot use `Compressed References` in regex when getting the date from java version output. In large heap, there is no `Compressed References` string.

```
23:58:50  OpenJDK Runtime Environment (build 11.0.7-internal+0-adhoc.****.BuildJDK11ppc64lelinuxxlNightly)
23:58:50  Eclipse OpenJ9 VM (build master-79aa8cfe4, JRE 11 Linux ppc64le-64-Bit 20200223_85 (JIT enabled, AOT enabled)
23:58:50  OpenJ9   - 79aa8cfe4
23:58:50  OMR      - 00c57af16
23:58:50  JCL      - 75dba088c45 based on jdk-11.0.7+3)
```